Assets/Scripts/Characters/Humans/Equipment/Weapon/Blades.cs

```diff
@@ -26,40 +27,54 @@ public override void PlayReloadAnimation()
             Hero.CrossFade(Hero.reloadAnimation, 0.1f);
         }
 
+        public override void EnableWeapons()
+        {
+            if (TotalBlades > 0)
+            {
+                WeaponLeft.SetActive(true);
+                WeaponRight.SetActive(true);
+                Hero.currentBladeSta = Hero.totalBladeSta;
+                bladesThrown = false;
+                Hero.weaponDisabledOnReloading = true;
+            }
+        }
```

Comment:
Since this enables the weapons, shouldn't this set Hero.weaponDisabledOnReloading to false instead of true?

Reply:
Yeah, I suppose I have the name backwards, should probably be called weaponEnabledAfterReloading, or invert the bool's usage

Reply:
Well even if you change the name, the boolean is still always true, so it might be unnecessary.